pkg/controller/masteruserrecord/masteruserrecord_controller_test.go

```diff
@@ -380,71 +380,116 @@ func TestModifyUserAccounts(t *testing.T) {
 }
 
 func TestSyncMurStatusWithUserAccountStatuses(t *testing.T) {
-	// given
 	logf.SetLogger(logf.ZapLogger(true))
 	s := apiScheme(t)
 
-	mur := murtest.NewMasterUserRecord("john",
-		murtest.StatusCondition(toBeNotReady(provisioningReason, "")),
-		murtest.AdditionalAccounts("member2-cluster", "member3-cluster"))
+	t.Run("mur status synced with updated user account statuses", func(t *testing.T) {
+		// given
+		mur := murtest.NewMasterUserRecord("john",
+			murtest.StatusCondition(toBeNotReady(provisioningReason, "")),
+			murtest.AdditionalAccounts("member2-cluster", "member3-cluster"))
+
+		userAccount := uatest.NewUserAccountFromMur(mur,
```

Comment:
could you refactor this code to something like https://github.com/codeready-toolchain/host-operator/pull/141/files#diff-93c8b7a9cc0875ddd6d3002d3d0b4253R460 which is more readable?

Reply:
It's the same old test I didn't change in the PR but sure... here we go: https://github.com/codeready-toolchain/host-operator/pull/141/commits/e81e9ec1667f125813afce6afa651df25f5bc768